### PR TITLE
Add batchCalls to LSP0 and LSP9

### PR DESCRIPTION
**Pull request by YamenMerhi**

### CODE REVIEW

File: LSPs/LSP-0-ERC725Account.md [hunk 1]
```diff
@@ -143,6 +143,33 @@ function renounceOwnership() external;
...
+
+```solidity
+function batchCalls(bytes[] calldata data) external returns (bytes[] memory results)
+```
+
+MUST batch calls on the contract by performing a delegatecall on the contract itself.
+
+_Parameters:_
+
+- `data`: the call data to be executed on the contract.
```
Comment (CJ42) [inline LSPs/LSP-0-ERC725Account.md:157]:
```suggestion
- `functionCalls`: a list of calldata to be executed on the contract.
```

File: LSPs/LSP-0-ERC725Account.md [hunk 2]
```diff
@@ -143,6 +143,33 @@ function renounceOwnership() external;
 This function is part of the [LSP14] specification.
 
 
+
+#### batchCalls
+
+```solidity
+function batchCalls(bytes[] calldata data) external returns (bytes[] memory results)
```
Comment (CJ42) [inline LSPs/LSP-0-ERC725Account.md:150]:
```suggestion
function batchCalls(bytes[] calldata functionCalls) external returns (bytes[] memory results)
```

File: LSPs/LSP-0-ERC725Account.md [hunk 3]
```diff
@@ -143,6 +143,33 @@ function renounceOwnership() external;
...
+- an array of ABI-encoded function calls such as an array of ABI-encoded execute, setData, transferOwnership or any LSP0 functions.
+- an array of bytes which will resolve to the fallback function to be checked for an extension. 
+
+
+_Requirements:_
+
+- MUST NOT be payable. 
+
+
+_Returns:_ `results` , an array of returned data of each called function.
```
Comment (CJ42) [inline LSPs/LSP-0-ERC725Account.md:170]:
```suggestion
_Returns:_ `results` , an array of data returned for each function called.
```

File: LSPs/LSP-11-BasicSocialRecovery.md [hunk 4]
```diff
@@ -260,7 +260,7 @@ MUST be emitted when the recovery process is finished by the controller who reac
 
 ### Setup
 
-In order to allow the social recovery contract to recover the linked target and add new permissions, the linked target should have an [LSP6-KeyManager](./LSP-6-KeyManager.md) as owner and the social recovery contract should have `ADDCONTROLLER` and `CHANGEPERMISSIONS` permissions set inside the **linked target** under this ERC725Y data key.
+In order to allow the social recovery contract to recover the linked target and add new permissions, the linked target should have an [LSP6-KeyManager](./LSP-6-KeyManager.md) as owner and the social recovery contract should have `ADDCONTROLLER` and `EDITPERMISSIONS` permissions set inside the **linked target** under this ERC725Y data key.
```
Comment (CJ42) [inline LSPs/LSP-11-BasicSocialRecovery.md:263]: good catch 👌

File: LSPs/LSP-9-Vault.md [hunk 5]
```diff
@@ -126,6 +126,32 @@ function renounceOwnership() external;
 This function is part of the [LSP14] specification.
 
 
+#### batchCalls
```
Comment (CJ42) [inline LSPs/LSP-9-Vault.md:129]: same comment as in LSP0 applies

**Review by CJ42 — APPROVED** [review]

**Review by frozeman — APPROVED** [review]

**Pull request merged**